js/src/views/Account/account.js

```diff
@@ -80,6 +82,15 @@ class Account extends Component {
     if (prevAddress !== nextAddress) {
       this.setVisibleAccounts(nextProps);
     }
+
+    const { isTestnet } = nextProps;
+    if (typeof isTestnet === 'boolean' && !this.state.verificationStore) {
+      const { api } = this.context;
+      const { address } = nextProps.params;
+      this.setState({
+        verificationStore: new VerificationStore(api, address, isTestnet)
```

Comment:
Is this related to https://github.com/ethcore/parity/pull/3665#discussion_r89991234 ?